gapic/samplegen/samplegen.py

```diff
@@ -278,7 +279,7 @@ def validate_and_transform_request(self,
                     # Python code can set protobuf enums from strings.
                     # This is preferable to adding the necessary import statement
                     # and requires less munging of the assigned value
-                    duplicate["value"] = f"'{val}'"
+                    # duplicate["value"] = f"'{val}'"
```

Comment:
what's up with this change...?

Reply:
Ah, that's cruft that needs to be removed. The same thing is now handled for all string values with the 

```python
if isinstance(duplicate["value"], str):
```